src/modules/Kdump.rb

```diff
@@ -1145,7 +1145,11 @@ def crash_kernel_values
       # modification.
       # The old value (ensuring the Array format) will be returned.
       if @crashkernel_list_ranges
-        return Array(@crashkernel_param_values.dup)
+        if @crashkernel_param_values.is_a? Symbol
```

Comment:
Sorry, I probably didn't explain myself well enough. With the suggested `to_s.dup`, the whole

```ruby
if @crashkernel_param_values.is_a? Symbol
 blah
else
 blah2
end
```
Wouldn't be needed because that `to_s.dup` works consistently for symbols and string. So the whole code section would look like.
```ruby
def crash_kernel_values
  # If the current values include "nasty" things and the user has not
  # overriden the value of @crashkernel_list_ranges to autorize the
  # modification.
  # The old value (ensuring the Array format) will be returned.
  if @crashkernel_list_ranges
    return Array(@crashkernel_param_values.to_s.dup)
  end

  result = []
  ....
```

And same below